### DISCUSSION TITLE
10mln records - advise

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#17081] [How to] [answered]
It happen that I will be uploading to custom module around 10 mln records (approx 200 chars each). What mysql config would you suggest for a such number?

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
check it out
https://github.com/YetiForceCompany/YetiForceCRM/blob/7961448df37bf133aba558f831e7c5cd046381b2/tests/setup/db/mysql.cnf#L55-L64
